website/docs/reference/appsmith-framework/context-object.md

```diff
@@ -9,33 +9,34 @@ description: >-
 
 The Appsmith object is a global object that provides access to information and functionalities within an application through objects and utility functions.
 
-The Appsmith object contains the following properties:
+## Properties
 
+The Appsmith object contains the following properties:
 
-## store `object`
+### store `object`
 
 <dd>
 
-This object contains the key-value pairs of the local storage of the app. You can add or update the values to the store using the [storeValue function](widget-actions/store-value.md). Conversely, you can access the values from the store using their corresponding keys, as shown below:
+This object contains the key-value pairs of the local storage of the app. You can add or update the values to the store using the [storeValue ()](widget-actions/store-value.md). Conversely, you can access the values from the store using their corresponding keys, as shown below:
```

Comment:
This object lets you access any app-level data or temporary state that is stored on the user's browser. You can add or update data using the storeValue() method. You can access saved data by referencing their keys: